### PR TITLE
Feature: Request certificate from school button

### PR DESCRIPTION
**Pull request by Siolto**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@nmshd/app-web",
-    "version": "2.3.10",
+    "version": "2.3.11",
```
Comment (jkoenig134) [inline package.json:3]: 2.4.0

File: package.json [hunk 2]
```diff
@@ -44,26 +44,26 @@
...
-        "@wdio/cli": "^8.15.0",
-        "@wdio/local-runner": "^8.15.0",
-        "@wdio/mocha-framework": "^8.15.0",
+        "@wdio/cli": "^8.15.5",
+        "@wdio/local-runner": "^8.15.4",
+        "@wdio/mocha-framework": "^8.15.4",
         "@wdio/spec-reporter": "^8.15.0",
         "babel-preset-transform-ui5": "^7.2.4",
-        "chromedriver": "^115.0.1",
+        "chromedriver": "^116.0.0",
```
Comment (vobu) [inline package.json:52]: `latest`?

File: src/core/controls/requests/items/FreeTextResponseItemRenderer.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            },
+            publicMethods: [],
+            events: {
+                change: { allowPreventDefault: true }
+            },
+            defaultAggregation: "_control"
+        },
+
+        init(e) {
+            const that = this
```
Comment (vobu) [inline src/core/controls/requests/items/FreeTextResponseItemRenderer.js:20]: rm var

File: src/core/controls/requests/items/FreeTextRequestItemRenderer.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            },
+            publicMethods: [],
+            events: {
+                change: { allowPreventDefault: true }
+            },
+            defaultAggregation: "_control"
+        },
+
+        init(e) {
+            const that = this
```
Comment (vobu) [inline src/core/controls/requests/items/FreeTextRequestItemRenderer.js:20]: rm `that`

File: src/flows/account/relationship/Home.controller.js [hunk 5]
```diff
@@ -85,6 +100,23 @@ sap.ui.define(
...
+                    "content.key": "AllowCertificateRequest",
+                    "content.value.@type": "ProprietaryBoolean"
+                }
+                const sharedToPeerAttributes = await this.getSharedToPeerAttributes(query)
+                for (const sharedToPeerAttribute of sharedToPeerAttributes) {
+                    if (sharedToPeerAttribute.value.value === true) {
+                        this.viewProp("/requestCertificateButtonVisible", true)
+                    }
+                }
+                console.log(sharedToPeerAttributes)
```
Comment (vobu) [inline src/flows/account/relationship/Home.controller.js:119]: rm

**Review by vobu — COMMENTED** [review]
small thingies only

File: package.json (hunk 2 above)
Comment (Siolto) [reply package.json:52]: that's what ncu proposed :)

**Review by vobu — APPROVED** [review]

**Pull request merged**